$NetBSD: patch-Makefile,v 1.3 2022/01/13 11:49:41 leot Exp $

- Instruct Makefile to properly handle also man pages and scripts via
  corresponding INSTALL_{MAN,SCRIPT} in a similar vein of INSTALL_BIN.
- Introduce MANDIR in order to ease the installation of man pages

--- Makefile.orig	2021-12-21 19:39:00.000000000 +0000
+++ Makefile
@@ -3,7 +3,10 @@ PERLSCRIPTS=vidir vipe ts combine zrun c
 MANS=sponge.1 vidir.1 vipe.1 isutf8.1 ts.1 combine.1 ifdata.1 ifne.1 pee.1 zrun.1 chronic.1 mispipe.1 lckdo.1 parallel.1 errno.1
 CFLAGS?=-O2 -g -Wall
 INSTALL_BIN?=install -s
+INSTALL_MAN?=install
+INSTALL_SCRIPT?=install
 PREFIX?=/usr
+MANDIR?=$(PREFIX)/man
 
 ifneq (,$(findstring CYGWIN,$(shell uname)))
 	DOCBOOKXSL?=/usr/share/sgml/docbook/xsl-stylesheets
@@ -26,10 +29,10 @@ isutf8: is_utf8/*.c is_utf8/*.h
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL_BIN) $(BINS) $(DESTDIR)$(PREFIX)/bin
-	install $(PERLSCRIPTS) $(DESTDIR)$(PREFIX)/bin
+	$(INSTALL_SCRIPT) $(PERLSCRIPTS) $(DESTDIR)$(PREFIX)/bin
 
-	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	install -m 644 $(MANS) $(DESTDIR)$(PREFIX)/share/man/man1
+	mkdir -p $(DESTDIR)$(MANDIR)/man1
+	$(INSTALL_MAN) $(MANS) $(DESTDIR)$(MANDIR)/man1
 
 uninstall:
 	for i in $(BINS) $(PERLSCRIPTS); do rm -f $(DESTDIR)$(PREFIX)/bin/$$i; done
